sw/airborne/arch/chibios/modules/lidar/vl53l5cx_api.c

```diff
@@ -0,0 +1,1311 @@
+/**
+  *
+  * Copyright (c) 2021 STMicroelectronics.
```

Comment:
These files are supposed to be platform independent. Maybe they should be in `peripherals` directory, on in the modules in non-arch part ?

Reply:
Ok, done.